
This just disables to build the 32bit support libs for now.

  - Rene Rebe <rene@exactcode.de>

--- ./gcc/config/i386/t-linux64.orig	2005-01-17 19:15:04.986820056 +0100
+++ ./gcc/config/i386/t-linux64	2005-01-17 19:15:46.038579240 +0100
@@ -4,9 +4,9 @@
 SHLIB_MAPFILES = $(srcdir)/libgcc-std.ver \
 		 $(srcdir)/config/i386/libgcc-x86_64-glibc.ver
 
-MULTILIB_OPTIONS = m64/m32
-MULTILIB_DIRNAMES = 64 32 
-MULTILIB_OSDIRNAMES = ../lib64 ../lib
+MULTILIB_OPTIONS = m64
+MULTILIB_DIRNAMES = 64
+MULTILIB_OSDIRNAMES = ../lib64 ../lib
 
 LIBGCC = stmp-multilib
 INSTALL_LIBGCC = install-multilib
--- ./gcc/config/sparc/t-linux64.orig	2005-01-17 19:20:21.991628000 +0100
+++ ./gcc/config/sparc/t-linux64	2005-01-17 19:22:44.576951736 +0100
@@ -1,6 +1,6 @@
-MULTILIB_OPTIONS = m64/m32
-MULTILIB_DIRNAMES = 64 32
-MULTILIB_OSDIRNAMES = ../lib64 ../lib
+MULTILIB_OPTIONS = m64
+MULTILIB_DIRNAMES = 64
+MULTILIB_OSDIRNAMES = ../lib64 ../lib
 
 LIBGCC = stmp-multilib
 INSTALL_LIBGCC = install-multilib





